xinference/model/llm/vllm/core.py

```diff
@@ -28,6 +29,8 @@
     Union,
 )
 
+from transformers import AutoTokenizer
```

Comment:
`transformers` is an optional dependency, so we may need to put the import into function.